apps/web/app/api/v1/management/surveys/[surveyId]/route.ts

```
@@ -110,6 +110,13 @@ export const PUT = async (
       }
     }
 
+    if (surveyUpdate.recaptcha && surveyUpdate.recaptcha.enabled) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)